### ISSUE TITLE
Keyboard trap when accessing rule on github.io

### ISSUE THREAD

**Issue by moekraft** [act-rules/act-rules.github.io#736]
Open rule https://act-rules.github.io/rules/2779a5

On Safari, cannot scroll page unless focus is put on collapsible <summary> 2.4.2. Page Titled. Once focus is put on this element, next tab goes to browser url field. 

On Firefox, it's a little better but trap still exists:
- Can tab to page and scroll, however, tabbing focus around page does not advance to elements on page. It's a tight loop from page url, to menu, to page, to 2.4.2 Page Titled and then back to url. Cannot access other elements on page.

Note: In both browsers, cannot scroll page via the scroll bar.

**Comment by jkodu** [comment]
Hi @moekraft 

It seems like both Safari and Firefox do not tab to elements/ links in a page by default.
Here is a link http://www.weba11y.com/blog/2014/07/07/keyboard-navigation-in-mac-browsers/ explaining how to configure this.

**Comment by WilcoFiers** [comment]
I think we misunderstood this issue. I looked at it a second time and can definitely see the issue here. I don't believe this is a WCAG issue, but we should solve it nevertheless.

There are two problems here:

1. The table of contents section sits on top of the scrollbar for the main content. Because of this, the main content doesn't have a visible scrollbar. This may cause people not to notice that the main content can be scrolled, and it makes scrolling more difficult for certain input devices

2. Because the main container isn't the one with the scrollbar, pressing `space` doesn't scroll the main content of the page like it normally would.

**Comment by moekraft** [comment]
I updated the Mac OSX keyboard navigation to tab to all controls, however, I still cannot tab to every element in the rule. The tab loop is very small and ends at the Accessibility Requirements Mapping section. 

I have recorded the screen, however, I cannot upload a .mov file to this issue.

**Comment by offsetChris** [comment]
Please re-open if the PR is not a sufficient solution or there are additional related issues.
